modules/core/src/main/java/org/apache/fluo/core/oracle/OracleServer.java

```diff
@@ -235,7 +239,7 @@ private synchronized long getTimestampsImpl(String id, int num) throws TExceptio
     }
 
     if (!isLeader) {
-      throw new IllegalStateException("Received timestamp request but Oracle is not leader");
+      throw new IllegalStateException("Received timestamp request but Oracle is not leader ");
```

Comment:
That's a weird whitespace to add.

Reply:
oh, I think I added something for debugging and then removed it.